src/lib/seam/components/AccessCodeTable/AccessCodeMainIcon.tsx

```diff
@@ -8,6 +8,10 @@ export interface AccessCodeMainIconProps {
   accessCode: AccessCode
 }
 
+function KeyIcon(): JSX.Element {
+  return <AccessCodeKeyIcon style={{ fontSize: '24px' }} />
+}
```

Comment:
This is another project convention: keep private functions below the main thing. So in this case, it would be moved to line 66.